plugins/wcx/unrar/src/unrarfunc.pas

```diff
@@ -25,6 +25,7 @@
 unit UnRARFunc;
 
 {$mode objfpc}{$H+}
+{$modeswitch arraytodynarray}
```

Comment:
When compiling with FPC 3.2.2, a Warning occurs here:

> Warning: Illegal compiler switch "ARRAYTODYNARRAY+"

You can ignore this, or I can write something like this:

```pascal
{$ifndef VER3_2}
{$modeswitch arraytodynarray}
{$endif}
```

Reply:
Yes, it is better to add IFDEF.

Reply:
Added IFDEF